contracts/TokenConverter/AbstractTokenConverter.sol

```diff
@@ -721,8 +847,13 @@ abstract contract AbstractTokenConverter is AccessControlledV8, IAbstractTokenCo
         uint256 tokenInUnderlyingPrice = priceOracle.getPrice(tokenAddressIn);
         uint256 tokenOutUnderlyingPrice = priceOracle.getPrice(tokenAddressOut);
 
+        uint256 incentive = configuration.incentive;
+        if (IConverterNetwork(converterNetwork).isTokenConverter(msg.sender)) {
```

Comment:
`converterNetwork` will be zero initially. In that case, I would keep the configured incentive